pyuvdata/uvdata/uvh5.py

```diff
@@ -524,8 +530,12 @@ def _read_header(
 
         # get frequency information
         self.freq_array = header["freq_array"][:, :]
-        self.channel_width = float(header["channel_width"][()])
         self.spw_array = header["spw_array"][:]
+        # We've added a new keyword, that did not exist before, so
```

Comment:
That was actually a comment that got left behind (and cut off, apparently) when I reorganized the code a little bit. I've moved the comment up to line 448, where its highlighting the fact that we need an extra bit of wrapping, because even though it's a required parameter, older versions of the UVH5 file format won't have this keyword set. Comments updated, pushed in 89a06d3.